packages/dai-plugin-migrations/src/migrations/SaiToDai.js

```diff
@@ -1,4 +1,4 @@
-import tracksTransactions from '@makerdao/dai/dist/src/utils/tracksTransactions';
```

Comment:
iirc webpack was unhappy with this way of importing from another package's file tree. My sense is that it might be a move in the right direction anyway, but let me know if you feel otherwise

Reply:
yeah, this was definitely a bit of a hack; ideally the tx-tracking logic would get pushed into its own little monorepo package